modules/ROOT/pages/io-connectors.adoc

```diff
@@ -0,0 +1,110 @@
+= Luna Streaming IO Connectors
+
+When you have Luna Streaming installed and running, add IO connectors to your deployment to interact with external systems, such as Apache Cassandra, Elastic Search, and many others.
+
+== Source Connectors
+
+Source connectors pull messages _from_ external topics and persist the messages _to_ Pulsar topics.
+
+Explore source connector configurations and code examples https://pulsar.apache.org/docs/en/io-connectors/#source-connector[here].
+
+== Sink Connectors
+
+Sink connectors are used to pull messages _from_ Pulsar topics and persist the messages _to_ external systems.
+
+Explore sink connector configurations and code examples https://pulsar.apache.org/docs/en/io-connectors/#sink-connector[here].
+
+== Processing Guarantees
+
+Processing guarantees are used to handle errors when writing messages to Pulsar topics.
+
+* *ATLEAST_ONCE*: Each message sent to the source can be processed more than once.
+* *ATMOST_ONCE*: The message sent to the source is processed at most once. Therefore, there is a chance that the message is not processed.
+* *EFFECTIVELY_ONCE*: Each message sent to the source will have one output associated with it.
+
+Learn more about processing guarantees https://pulsar.apache.org/docs/en/io-overview/#processing-guarantee[here].
+
+== Kafka Connectors
+
+Datastax offers https://github.com/datastax/pulsar-3rdparty-connector[third-party connector support] for creating Pulsar connectors on the base of existing Kafka connectors. This allows developers to quickly move connectors from their Apache Kafka infrastructure into Apache Pulsar.
+
+**Prerequisites:**
+
+* Ensure you have **JDK 11+** and **Maven 3.8** installed.
+
+* Clone the connector's repo and `run mvn clean install` from the root.
+
+* The connector's `.nar` files can be found at `pulsar-connectors/<connector name>>/target/pulsar-3rdparty-pulsar-connectors-<connector name>-0.1.0-SNAPSHOT.nar`.
+
+=== Configuring connectors
+
+Follow the documentation https://pulsar.apache.org/docs/en/io-use/[here] to initialize your connector. Then, modify its `config.yaml` file as below to create an adapter for the connector. For further information on shading and troubleshooting, see the https://github.com/datastax/pulsar-3rdparty-connector[GitHub readme].
```

Comment:
@mendonk Let's give them enough basic steps they can do it here. Then we can point to the pulsar docs for more info.